edb/lib/std/25-setoperators.eql

```diff
@@ -21,34 +21,23 @@
 ## --------------------------
 
 
-CREATE INFIX OPERATOR
-std::`IN` (e: anytype, s: SET OF anytype) -> std::bool
-    FROM SQL EXPRESSION;
-
+# The set membership operators (IN, NOT IN) are defined
+# in terms of the corresponding equality operator.
 
 CREATE INFIX OPERATOR
-std::`IN` (e: decimal, s: SET OF float64) -> std::bool
-    FROM SQL EXPRESSION;
-
-
-CREATE INFIX OPERATOR
-std::`IN` (e: float64, s: SET OF decimal) -> std::bool
+std::`IN` (e: anytype, s: SET OF anytype) -> std::bool
+{
     FROM SQL EXPRESSION;
+    SET derivative_of := 'std::=';
+};
 
 
 CREATE INFIX OPERATOR
 std::`NOT IN` (e: anytype, s: SET OF anytype) -> std::bool
+{
     FROM SQL EXPRESSION;
-
-
-CREATE INFIX OPERATOR
-std::`NOT IN` (e: decimal, s: SET OF float64) -> std::bool
-    FROM SQL EXPRESSION;
-
-
-CREATE INFIX OPERATOR
-std::`NOT IN` (e: float64, s: SET OF decimal) -> std::bool
-    FROM SQL EXPRESSION;
+    SET derivative_of := 'std::=';
```

Comment:
Is there a reason why `NOT IN` is derived from `=` as opposed to `!=`? It seems slightly odd.